community/go/APKBUILD

```diff
@@ -3,7 +3,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=go
 # go binaries are statically linked, security updates require rebuilds
-pkgver=1.13
+pkgver=1.13.1
```

Comment:
also "secfixes" needs to add https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-16276